pkg/repository/postgres/mq.go

```diff
@@ -55,17 +55,7 @@ func (m *messageQueueRepository) Listen(ctx context.Context, name string, f func
 
 	l.Handle(name, handler)
 
-	err := l.Listen(ctx)
-
-	if err != nil {
-		if errors.Is(err, context.Canceled) {
-			return nil
-		}
-
-		return err
-	}
-
-	return nil
+	return l.Listen(ctx)
```

Comment:
this was just changed too I think: https://github.com/hatchet-dev/hatchet/pull/1631/files#diff-7370cadfe1fad4544664f60ea1b590865daa2c10b6225467b3af38bd5b6717dcR58